### PR TITLE
Revision update

### PR DESCRIPTION
**Pull request by johli**
### Description of your changes

- Added new sh script to setup multi-fold data directories (TFRecord symlinks).
- Updated all sh scripts (data processing / training / eval / QTLs / CRISPR).
- Updated scripts / notebooks for aggregating replicate gradient scores for fold 3 (multiple crosses).
- Moved data processing scripts/Makefile for published model from 'borzoi' repo to this repo.
- Added sub-directory READMEs with more documentation about model training / data processing / evaluation.
- Updated README with recommended usage instructions.

### Type of change
- [X] Bug fix
- [X] New feature
- [X] Refactoring / code clean-up
- [X] Documentation add / update

### (If applicable) How has this been tested?

Tests:
- Re-ran all scripts from 'borzoi-paper' repo and 'borzoi' tutorials.
- Checked that all major results from the paper reproduced (test evals, e/s/paQTLs).

Environment:
- Python 3.10
- TensorFlow 2.15

### CODE REVIEW

**Review by davek44 — APPROVED** [review]
I'm just going to trust you on this one :)
Let me know if there's anything in particular you want me to look at.
